src/globus_sdk/services/transfer/client.py

```diff
@@ -1243,6 +1250,17 @@ def operation_ls(
         :param query_params: Additional passthrough query parameters
         :type query_params: dict, optional
 
+        .. note::
+
+            Pagination is not supported by the GridFTP protocol, and therefore
+            limit+offset pagination will result in the Transfer service repeatedly
+            fetching an entire directory listing from the server and filtering it before
+            retuning it to the client.
+
+            Such usage may still be more efficient than asking for a very large directory
+            listing, for latency-sensitive applications, as it reduces the size of the payload
```

Comment:
```suggestion
            listing for latency-sensitive applications, as it reduces the size of the payload
```

Reply:
Alternatively, the "for latency-sensitive applications" line could be the opening to the sentence, but that was ugly to rewrite as a suggestion in this interface.

> For latency-sensitive applications, such usage may still ...

Reply:
I was hesitating on this one because it felt awkward once I was looking at it.
I agree with the `For ...`-first rewrite -- it reads best. I'll apply that momentarily.